cmake/edge_configure.cmake

```diff
@@ -109,9 +109,23 @@ if (${FIRMWARE_UPDATE})
   MESSAGE ("Enabling firmware update for Edge")
 
   if (FOTA_ENABLE)
-    execute_process ( COMMAND cp ${CMAKE_CURRENT_SOURCE_DIR}/fota/fota_install_callback.c ${CMAKE_CURRENT_SOURCE_DIR}/lib/mbed-cloud-client/fota/ )
+    execute_process ( COMMAND cp ${CMAKE_CURRENT_SOURCE_DIR}/fota/fota_app_callbacks.c ${CMAKE_CURRENT_SOURCE_DIR}/lib/mbed-cloud-client/fota/ )
+    if (NOT DEFINED FOTA_SCRIPT_DIR)
+      SET (FOTA_SCRIPT_DIR \"/opt/pelion\")
+    endif()
+    if (NOT DEFINED FOTA_INSTALL_MAIN_SCRIPT)
+      SET (FOTA_SCRIPT_DIR \"fota_update_activate.sh\")
+    endif()
+
     add_definitions(-DFOTA_DEFAULT_APP_IFS=1)
     add_definitions(-DTARGET_LIKE_LINUX=1)
+    add_definitions(-DFOTA_SCRIPT_DIR=${FOTA_SCRIPT_DIR})
+    add_definitions(-DFOTA_INSTALL_MAIN_SCRIPT=${FOTA_INSTALL_MAIN_SCRIPT})
+    if (FOTA_COMBINED_IMAGE_SUPPORT) 
```

Comment:
@davidsaada if the user wants this feature then they have to build with `-DFIRMWARE_UPDATE=ON` `-DFOTA_ENABLE=ON` and `-DFOTA_COMBINED_IMAGE_SUPPORT=ON`, correct?

Reply:
Yes. I will create a separate PR for meta-mbed-edge that will include these (currently only for uz boards).